cookiecutter-xblock/{{cookiecutter.repo_name}}/{{cookiecutter.package_name}}/{{cookiecutter.package_name}}.py

```diff
@@ -1,11 +1,19 @@
 """TO-DO: Write a description of what this XBlock is."""
 
-import pkg_resources
+import os
+from importlib import resources
+
 from django.utils import translation
 from web_fragments.fragment import Fragment
 from xblock.core import XBlock
 from xblock.fields import Integer, Scope
-from xblockutils.resources import ResourceLoader
+
+try:
+    from xblock.utils.resources import ResourceLoader
+except ModuleNotFoundError:
+    from xblockutils.resources import ResourceLoader
```

Comment:
I don't think we need to support `xblockutils` in cookiecutter. All new projects should be using `xblock.utils`

Reply:
+1 , we don't need to provide backward support in cookie cutter

Reply:
Sure. I'll drop the backward compatibility support.